corehq/apps/zapier/tests.py

```diff
@@ -74,7 +81,9 @@
 </data>
 """ % FORM_XMLNS
 ZAPIER_URL = "https://zapier.com/hooks/standard/1387607/5ccf35a5a1944fc9bfdd2c94c28c9885/"
+TEST_URL = "http://commcarehq.org/?domain=johto&case_type=teddiursa"
```

Comment:
nit: since these are only used in a single test, you could make them instance properties of that test class, so that you don't have to scroll all the way up here to see what the values are. 
i.e.
```py
class TestZapierCustomFields(TestCase):
    def setUp(self):
        super(TestZapierCustomFields).setUp()
        self.test_url = "..."
```